src/common/wrap/index.js

```diff
@@ -1,46 +1,14 @@
 
-import {wrapFetch as wf} from './wrap-fetch'
-import {wrapTimer as wt} from './wrap-timer'
-import {wrapRaf as wr} from './wrap-raf'
-import {wrapHistory as wh} from './wrap-history'
-import {wrapJsonP as wj} from './wrap-jsonp'
-import {wrapMutation as wm} from './wrap-mutation'
-import {wrapPromise as wp} from './wrap-promise'
-import {wrapXhr as wx} from './wrap-xhr'
-import {wrapEvents as we} from './wrap-events'
-
-export function wrapEvents(sharedEE) {
-  return we(sharedEE)
-}
-
-export function wrapFetch(sharedEE) {
-  return wf(sharedEE)
-}
-
-export function wrapHistory(sharedEE) {
-  return wh(sharedEE)
-}
-
-export function wrapJson(sharedEE) {
-  return wj(sharedEE)
-}
-
-export function wrapMutation(sharedEE) {
-  return wm(sharedEE)
-}
-
-export function wrapPromise(sharedEE) {
-  return wp(sharedEE)
-}
-
-export function wrapRaf(sharedEE) {
-  return wr(sharedEE)
-}
-
-export function wrapTimer(sharedEE) {
-  return wt(sharedEE)
-}
-
-export function wrapXhr(sharedEE) {
-  return wx(sharedEE)
-}
+import {wrapFetch, unwrapFetch} from './wrap-fetch'
+import {wrapTimer, unwrapTimer} from './wrap-timer'
+import {wrapRaf, unwrapRaf} from './wrap-raf'
+import {wrapHistory, unwrapHistory} from './wrap-history'
+import {wrapJsonP, unwrapJsonP} from './wrap-jsonp'
+import {wrapMutation, unwrapMutation} from './wrap-mutation'
+import {wrapPromise, unwrapPromise} from './wrap-promise'
+import {wrapXhr, unwrapXhr} from './wrap-xhr'
+import {wrapEvents, unwrapEvents} from './wrap-events'
+
+export {wrapEvents, wrapFetch, wrapHistory, wrapJsonP, wrapMutation, wrapPromise, wrapRaf, wrapTimer, wrapXhr};
+
+export {unwrapEvents, unwrapFetch, unwrapHistory, unwrapJsonP, unwrapMutation, unwrapPromise, unwrapRaf, unwrapTimer, unwrapXhr};
```

Comment:
🤔 maybe combine these two exports into a single export? Or maybe use `export {wrapFetch, unwrapFetch} from './wrap-fetch'`. Personally, I hate barrel files like this because they can turn into points of circular reference leading to terrible errors to debug. However, it was already here. I think we should add a webpack plugin to detect circular dependencies if we can.

Reply:
Combined